This patch for festival for Debian GNU/Linux, configures
additional russian language loading support from festival command line

Index: festival/lib/languages.scm
===================================================================
--- festival.orig/lib/languages.scm	2009-09-12 18:00:44.000000000 -0500
+++ festival/lib/languages.scm	2009-09-12 18:01:40.000000000 -0500
@@ -101,6 +101,14 @@
    (Parameter.set 'Language 'czech)
 )
 
+(define (language_russian)
+"(language_russian)
+Set up language parameters for Russian."
+  (set! male1 voice_msu_ru_nsh_clunits)
+  (male1)
+  (Parameter.set 'Language 'russian)
+)
+
 (define (language_scots_gaelic)
 "(language_scots_gaelic)
 Set up language parameters for Scots Gaelic."
@@ -150,6 +158,8 @@
     (language_czech))
    ((equal? language 'klingon)
     (language_klingon))
+   ((equal? language 'russian)
+    (language_russian))
    (t
     (print "Unsupported language, using English")
     (language_british_english))))
Index: festival/src/main/festival_main.cc
===================================================================
--- festival.orig/src/main/festival_main.cc	2009-09-12 18:00:44.000000000 -0500
+++ festival/src/main/festival_main.cc	2009-09-12 18:00:45.000000000 -0500
@@ -108,7 +108,7 @@
 	"              are printed (default if stdin not a tty)\n"+
 	"--language <string>\n"+
         "              Run in named language, default is\n"+
-	"              english, spanish and welsh are available\n"+
+	"              english, spanish, russian and welsh are available\n"+
 	"--server      Run in server mode waiting for clients\n"+
 	"              of server_port (1314)\n"+
 	"--script <ifile>\n"+
